build.gradle

```diff
@@ -22,6 +22,8 @@ allprojects {
 
     repositories {
         jcenter()
+        mavenLocal()
```

Comment:
I think we could not until new Marathon lib version will be released, cause you've already patched Marathon client. So, remove that for passing travis tests and I merge your PR.